src/main/src/main/java/org/geoserver/catalog/ResourcePool.java

```diff
@@ -1381,6 +1388,13 @@ else if (ppolicy == ProjectionPolicy.NONE && nativeCRS != null) {
                 schema = typeBuilder.buildFeatureType();
             }
 
+            // applying wrappers using
+            // implementations of org.geoserver.catalog.RetypeFeatureTypeCallback
```

Comment:
Please shorten this comment, if the class is moved or renamed this will be come just noise.

Reply:
will do